app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```

Comment:
```suggestion
    def __init__(self, 
    comfort_class: int,
```
it would be more readable to move it to the next line